cross/stockfish/Makefile

```diff
@@ -0,0 +1,46 @@
+PKG_NAME = sf
```

Comment:
Please name package as "stockfish" even if upstream only archive is "sf". You may use `PKG_DIST_FILE` to do so.